### PR TITLE
chore(@storybook): add about view to storybook, refactor about view to use calls from store instead of global

### PR DESCRIPTION
**Pull request by iurimatias**
### What does the PR do

Screen Shot 2022-10-13 at 2 46 24 PM

### Affected areas

AboutView.qml

### CODE REVIEW

**Review by caybro — APPROVED** [review]
Looks good but isn't the view center-aligned in status-desktop?

**Comment by caybro** [comment]
@micieslak we could also "fake" the `Qt.application.displayName/version` :)

**Review by micieslak — CHANGES_REQUESTED** [review]

File: ui/app/AppLayouts/Profile/views/AboutView.qml [hunk 1]
```diff
@@ -99,7 +100,7 @@ SettingsContentBase {
                 text: qsTr("Status Desktop")
                 icon.width: 0
                 onClicked: {
-                    Global.openLink("https://github.com/status-im/status-desktop")
+                    root.store.openLink("https://github.com/status-im/status-desktop")
```
[the remark this answers is not in the document]
Comment (caybro) [reply ui/app/AppLayouts/Profile/views/AboutView.qml:103]: In general I agree with the proposed solution; however we should strive for making AppMain smaller, not bigger :)

[the remark this answers is not in the document]
Comment (micieslak) [reply ui/app/AppLayouts/Profile/views/AboutView.qml:103]: Yeah, point 6 is about that, the same can be done with e.g. popups that currently live in AppMain

[the remark this answers is not in the document]
Comment (caybro) [reply ui/app/AppLayouts/Profile/views/AboutView.qml:103]: Yup, I hate e.g. the way we drag the emoji popup around everywhere

[the remark this answers is not in the document]
Comment (micieslak) [reply ui/app/AppLayouts/Profile/views/AboutView.qml:103]: Ok, so for now we have the component adjusted to run in storybook, it works. For further changes there is a separate ticket: https://github.com/status-im/status-desktop/issues/7933

**Review by micieslak — APPROVED** [review]
I'm happy to see storybook used!

File: ui/app/AppLayouts/Profile/views/AboutView.qml (hunk 1 above)
[the remark this answers is not in the document]
Comment (noeliaSD) [reply ui/app/AppLayouts/Profile/views/AboutView.qml:103]: I do agree with your solution. Great proposal!

**Pull request merged**